core/misc/zen-ci/run_phpcs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+##
+# Run phpcs code standards tests.
+##
+
+#set site path
```

Comment:
space, capital, period

Reply:
addressed in latest commit. ad6a5a785aa61a73c219dfcbee119adb340a2cd8